index.html

```diff
@@ -0,0 +1,209 @@
+
+            <p>Severní pobřeží je pak místy opravdová divočina, už tak mizivé osídlení ještě více prořídne a vy projíždíte místní fjordy po šotolinových cestách, aniž byste hodiny potkali jediné auto. Nad hlavou vám ční vysoké hory pokryté věčným sněhem,
+                každým údolím se v kaskádách a vodopádech řítí dolů řeky, prostě divočina zamrznutá v čase. Dostane se vám to tu pod kůži a nutí Vás to přemýšlet, jak je ve skutečnosti člověk maličký a nicotný&hellip;</p>
+
+
+            <img src="images/island1.jpg" alt="Island">
+            <p class="popisek">
+                Photo &copy; <a href="https://flic.kr/p/yiAg2h">Moyan Brenn</a>
+            </p>
+
+        </section>
+        <!-- sekce Co musíte vidět -->
+        <section class=text2>
+            <h2 id="co">Co musíte vidět</h2>
+            <hr>
+            <p><strong>Poloostrov Reykjanes</strong> - Než z letiště v Keflavíku zamíříte do hlavního města, zajeďte se podívat na tento poloostrov. Je to zajížďka jen pár kilometrů a rázem se dostanete do krajiny pro Island přímo typické. Cestu lemují lávová
+                pole připomínající doslova měsíční krajinu. Pro Středoevropana něco neuvěřitelného. Ani kousek zeleně, kam oko dohlédne jen černá suť a rozpraskané lávové desky. Mezi tím vším až mysteriózně vypadající geotermální elektrárny. Na pobřeží
+                pak vystupují z vody poutavé skalní útvary Reykjanesta.</p>
+
+            <p><strong>Reykjavík</strong> - Hlavní město určitě nevynecháte kvůli doplnění proviantu. Za zmínku stojí zajímavě architektonicky řešený kostel Hallgrímskirkja, centrum města a přístav se svou výstavbou začínají podobat všem vyspělým velkoměstům
+                v Evropě a příliš milovníky severu neosloví. Zavítáte-li však do spleti uliček pod katedrálou, nebudete litovat, ty už atmosféru mají.</p>
+
+            <p><strong>Gullfoss</strong> - Vodopádové menu začínáme u kaskádovitého vodopádu na řece Hvítá. Postavit se na hranu mezi kaskádami a hledět do té masy vody, řítící se do kaňonu pod vámi je vážně zážitek&hellip;</p>
+
+            <p><strong>Geysir</strong> - Kousek od Gullfossu je oblast, která turisty přitahuje jako magnet. Geysir, který dal název všem ostatním gejzírům světa. On sám se však již téměř odmlčel, údajně chrlí vodu jen velmi zřídka. Zato jeho menší a mladší
+                soused, pojmenovaný Strokkur, prská do okolí vařící sloupec vody v méně či více pravidelných intervalech každých pár minut.</p>
+
+            <p><strong>Seljalandsfoss</strong> - To už jsme na jižním pobřeží. Tento 60 metrů vysoký vodopád je výjimečný především tím, že se na něho můžete podívat nejen shora, ze stran, ale také jej obejít a hledět do okolní krajiny přes masy vody, které
+                kolem Vás padají.</p>
+
+            <img src="../../images/island3.jpg" alt="Island">
```

Comment:
Ahoj, tady a u dalších bych dala přednost zdroji jen "images/island3.jpg" což úplně stačí protože je složka images na úrovni kódu (index.html), zatímco tento zápis, který jde o dvě úrovně výš (../ je jedna úroveň výš), když bys pak chtěla používat takle relativní cestu, můžeš napsat "./images/island3.jpg" - tím půjdeš na začátek projektu a taky ti to zafunguje :)